magento/magento-cloud-docker
# MAGECLOUD-3248: Add control over docker mounts

### Description

Add control over Docker mounts.

### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3248

### Manual testing scenarios

1. Clone a cloud template.
2. Make the following changes to the `composer.json` file.
```
    "repositories": {
        "repo": {
            "type": "composer",
            "url": "https://repo.magento.com"
        },
        "ece-tools": {
            "type": "vcs",
            "url": "https://github.com/magento/ece-tools"
        },
        "docker": {
            "type": "vcs",
            "url": "https://github.com/magento/magento-cloud-docker"
        }
    },
    "require": {
        "magento/magento-cloud-metapackage": ">=2.3.2 <2.3.3",
        "magento/ece-tools": "dev-develop as 2002.0.99",
        "magento/magento-cloud-docker": "dev-MAGECLOUD-3248 as 1.0.99"
    }
```
3. Run `composer update`.
4. Generate the docker configuration file `docker-compose.yml` with `./vendor/bin/ece-docker build:compose`.
5. Launch docker with `docker-compose up -d`.
6. Build and deploy Magento in the container with `docker-compose run build cloud-build` and `docker-compose run deploy cloud-deploy` respectively.
7. Access local Magento (https://magento2.docker/) instance without errors.
8. Inspect the web container `docker inspect <project>_web_1` (for example) and confirm that the `mounts` defined in the `.magento.app.yaml` of the project are present in the container `Mounts`.

### Contribution checklist
